### PR TITLE
Feature/creating a lecture leads not to the classroom but to the lecturers view

### PR DESCRIPTION
**Pull request by GerritSe**

### CODE REVIEW

File: client/views/lecture/lecture_page.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ Template.lecturePage.helpers({
     return questions;
   },
 
+  lectureDataReady: function() {
+    lectureData = Lectures.findOne({lectureCode: this.lectureCode});
+    if(lectureData)
```
Comment (coreyhaines) [inline client/views/lecture/lecture_page.js]: Reminder that it is a good practice to use `{ }` on single-line if clauses

File: client/views/question/question.js [hunk 2]
```diff
@@ -47,5 +58,17 @@ Template.question.events({
       if(error)
         return alert(error.reason);
     });
-  }
+  },
+  'click .btn-delete-question': function () {
+    
+    var questionDeleteParams = {
+    questionId: this._id,
```
Comment (coreyhaines) [inline client/views/question/question.js]: indentation will make this easier to read.

File: client/views/question/question.js [hunk 3]
```diff
@@ -47,5 +58,17 @@ Template.question.events({
...
+    lectureCode: this.lectureCode
+    }
```
Comment (coreyhaines) [inline client/views/question/question.js]: This is the end of a `var`, so should have semi-colon

File: lib/collections/lectures.js [hunk 4]
```diff
@@ -31,8 +32,7 @@ Meteor.methods({
       var userId = Meteor.userId();
       Lectures.update({lectureCode:lectureCode}, {$pull: {members: {$in: [userId]}}});
     }
```
Comment (coreyhaines) [inline lib/collections/lectures.js]: end of the `return` should have a semi-colon

File: lib/collections/questions.js [hunk 5]
```diff
@@ -27,16 +27,32 @@ Meteor.methods({
...
+        if(error){
+          return alert(error.reason);
+        }
+      });  
+    }
+  },
+
+  deleteQuestion: function(questionDeleteParams){
+    
+    lectureCode = questionDeleteParams.lectureCode;
```
Comment (coreyhaines) [inline lib/collections/questions.js]: missing `var`

**Comment by minden** [comment]
@coreyhaines Thank you very much for the feedback. I have fixed the issues in this and the following branch. 
https://github.com/minden/GotIt/pull/28

**Pull request merged**